tlatools/org.lamport.tlatools/src/tla2sany/drivers/SANY.java

```diff
@@ -382,13 +382,27 @@ private static void printUsage()
       final List<List<UsageGenerator.Argument>> commandVariants = new ArrayList<>();
       final List<UsageGenerator.Argument> variant = new ArrayList<>();
       variant.add(new UsageGenerator.Argument(
-          "-s", "Turns off semantic analysis", true));
+          "-s", "Turns off semantic analysis and level-checking.", true));
       variant.add(new UsageGenerator.Argument(
-          "-l", "Turns off level checking. Level checking won't be\n" +
-          "used, if the semantic analysis is disabled. ", true));
+          "-l", "Turns off level-checking.", true));
       variant.add(new UsageGenerator.Argument(
-          "-error-codes", "If enabled, error level will be reported as the tools'\n" +
-          "return value." , true));
+          "-error-codes", "Return a descriptive exit code in case of error.\n\n" +
+          "'1' Error during initialization.\n" +
+          "'2' Error during parsing.\n" +
+          "'3' Error parsing built-in operator tables.\n" +
+          "'4' Error during semantic analysis or level-checking.", true));
```

Comment:
I am pretty sure the possibility of an error occurring during either initialization or parsing built-in operator tables (or both) was removed by #1012. So we might not want to document these.

Reply:
Yeah I confirmed it's impossible for these errors to ever occur, although the error reporting machinery still exists. I took out this machinery in a new PR #1069.

Reply:
I've removed the 2 cases.